ddtrace/contrib/util.py

```diff
@@ -7,7 +7,7 @@
 
 deprecation(
     name="ddtrace.contrib.util",
-    message="Use `ddtrace.utils.importlib` module instead",
+    message="Use `ddtrace.internal.utils.importlib` module instead",
```

Comment:
Missed this last one

Reply:
should we be suggesting `ddtrace.internal.` utils?

should we keep as `ddtrace.utils` here?

Reply:
```suggestion
    message="Use `ddtrace.utils.importlib` module instead",
```